locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: 0.1.1\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2021-05-20 14:40-0700\n"
+"POT-Creation-Date: 2021-05-23 10:45+0000\n"
 "PO-Revision-Date: 2021-04-05 12:44+0100\n"
 "Last-Translator: Fabien Basmaison <contact@arkhi.org>\n"
```

Comment:
I definitely use git blame rather than this metadata, I think it's standard on this filetype because these types of files aren't always part of git. It's very much a thing that's here because the script that generated the file put it there